web/router.go

```diff
@@ -73,6 +73,9 @@ func serverEngine(app *services.ChainlinkApplication) *gin.Engine {
 		v2.GET("/backup", backup.Show)
 	}
 
+	cc := ConfigController{app}
+	engine.GET("/config", cc.Show)
```

Comment:
I personally would version config `/v2/config`

Reply:
👍 I originally thought the same thing but the story asks for `routing: GET request to /config`. Thoughts @smartcontractkit/core-devs?

Reply:
We got the greenlight for `/v2/config`, let's change it.